### DISCUSSION TITLE
no such file or directory, lstat '/vercel/path0/vercel/path0/.next/routes-manifest.json'

### DISCUSSION THREAD

**Discussion by craigboldy** [vercel/community#2749] [Help]
### Summary

When deploying my next.js app it fails with the following errors 
14:08:37.794 | Error: ENOENT: no such file or directory, lstat '/vercel/path0/vercel/path0/.next/routes-manifest.json'
14:08:37.859 | ENOENT: ENOENT: no such file or directory, lstat '/vercel/path0/vercel/path0/.next/routes-manifest.json'

Deploy Log Below 

Route (pages)                              Size     First Load JS
14:08:34.192 | ┌   /_app                                  0 B            78.4 kB
14:08:34.192 | ├ λ /[[...slug]]                           3.13 kB          95 kB
14:08:34.192 | ├ ○ /404                                   194 B          78.6 kB
14:08:34.192 | ├ λ /api/preview-mode                      0 B            78.4 kB
14:08:34.192 | └ λ /blog/[...slug]                        3.13 kB          95 kB
14:08:34.192 | + First Load JS shared by all              79.6 kB
14:08:34.192 | ├ chunks/framework-4556c45dd113b893.js   45.2 kB
14:08:34.192 | ├ chunks/main-a511c91c5de587c8.js        28.5 kB
14:08:34.192 | ├ chunks/pages/_app-54479a597fe3bd27.js  2.56 kB
14:08:34.192 | ├ chunks/webpack-b6b60be90eeda833.js     2.13 kB
14:08:34.192 | └ css/5e9e514a2d9f2ba6.css               1.14 kB
14:08:34.193 |  
14:08:34.193 | λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
14:08:34.193 | ○  (Static)  automatically rendered as static HTML (uses no initial props)
14:08:34.193 |  
14:08:34.812 | Done in 25.56s.
14:08:37.792 | Traced Next.js server files in: 2.956s
14:08:37.794 | Error: ENOENT: no such file or directory, lstat '/vercel/path0/vercel/path0/.next/routes-manifest.json'
14:08:37.859 | ENOENT: ENOENT: no such file or directory, lstat '/vercel/path0/vercel/path0/.next/routes-manifest.json'

I've tried adding `VERCEL_FORCE_NO_BUILD_CACHE` `1` as mentioned in another ticket but no joy. The site builds fine on my local setup also.

**Comment by andrewabril** [comment]
I'm getting the same error on my projects as well. It's a turbo repo with a Nextjs and sanity.io projects. They started failing with this same error tonight, when just a couple days ago, it was working fine. I even tried **re-deploying the latest working deployment** in both Preview and Production, and get the same errors. Additionally, all my deployments triggered by new commits are automatically canceled with the default `npx turbo-ignore` setting. I have to redeploy it without ignore step to then get the manifest error.

I've tried:
1. Removing the ignore step to actually trigger builds
2. Downgrading Nextjs from 13.3 to 13.1.1 with no luck
3. Redeploying without cache and ignore step

I'm not sure where to go from here - it works perfectly locally, and I can build both apps without any issues. I haven't changed the project settings other than remove the build ignore step, and are using the default build, pnpm install, etc. commands.

**Comment by andrewabril** [comment]
For anyone having the same issue - the issue for me was my turbo repo having two lock files: pnpm and yarn. Deleting the yarn lockfile (I'm using pnpm) resolved it. Not sure why it didn't cause an issue the past 4 months, but look out for it!

Shout out to @anthonyshew for solving the issue.

Thank you!

**Comment by craigboldy** [comment]
If I downgrade next.js to `12.3.4` then it works, any 13 releases doesn't. I don't have multiple .lock files tried everything I can see in other similar issues with not joy.
